FiveCalls/FiveCalls/Dashboard.swift

```diff
@@ -48,26 +48,11 @@ struct Dashboard: View {
             .onAppear() {
 //              TODO: refresh if issues are old too?
                 if appState.issues.isEmpty {
-                    op.fetchIssues(delegate: (UIApplication.shared.delegate as! AppDelegate), completion: { result in
-                        if case let .serverError(error) = result {
-                            print("issues error: \(error)")
-                        }
-                        if case .offline = result {
-                            print("issues offline")
-                        }
-                    })
+                    op.fetchIssues(setIssues: appState.setIssues)
                 }
         
                 if let location = appState.location, appState.contacts.isEmpty {
-                    op.fetchContacts(location: location, delegate: (UIApplication.shared.delegate as! AppDelegate)) { result in
-                        if case let .serverError(error) = result {
-                            print("contacts error: \(error)")
-                        }
-                        if case .offline = result {
-                            print("contacts offline")
-                        }
-        
-                    }
+                    op.fetchContacts(location: location, fetching: appState.setFetchingContacts, setContacts: appState.setContacts)
```

Comment:
Likewise, just allowing access to the app state from these fetch operations, we could avoid passing in the handlers.